Content.Server/Xenoarchaeology/XenoArtifacts/Effects/Systems/GravityWellArtifactSystem.cs

```diff
@@ -0,0 +1,51 @@
+using Content.Server.Singularity.Components;
+using Content.Server.Singularity.EntitySystems;
+using Content.Server.Xenoarchaeology.XenoArtifacts.Effects.Components;
+using Robust.Shared.Random;
+
+namespace Content.Server.Xenoarchaeology.XenoArtifacts.Effects.Systems;
+
+public sealed class GravityWellArtifactSystem : EntitySystem
+{
+    #region Dependencies
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] private readonly GravityWellSystem _gravWellSystem = default!;
+    #endregion Dependencies
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<GravityWellArtifactComponent, ComponentStartup>(SetupGravityWell);
+        SubscribeLocalEvent<GravityWellArtifactComponent, ComponentShutdown>(ShutdownGravityWell);
+    }
+
+    private void SetupGravityWell(EntityUid uid, GravityWellArtifactComponent comp, ComponentStartup args)
```

Comment:
Use `ArtifactNodeEnteredEvent` instead of componentstartup.

Reply:
Done.